app/bebbo/dynamicImports.ts

```diff
@@ -18,3 +27,31 @@ export const standardDevData =require('@offlinedata/bebbo/standardDevData').stan
 export const SurveyData =require('@offlinedata/bebbo/SurveyData').SurveyData;
 export const vaccineData =require('@offlinedata/bebbo/vaccineData').vaccineData;
 export const VideoArticleData =require('@offlinedata/bebbo/VideoArticleData').VideoArticleData;
+import northMacedinaPartnerLogo from '@assets/appOfflineImages/northmacedonia/partner/1.png';
+export const nmPartnerLogo=northMacedinaPartnerLogo;
+import northMacedinaSponserLogo from '@assets/appOfflineImages/northmacedonia/sponser/2.png';
+export const nmSponserLogo=northMacedinaSponserLogo;
+import albeniaPartnerLogo from '@assets/appOfflineImages/albenia/partner/IPH.png';
+export const albeniaPartnerLogo1=albeniaPartnerLogo;
+import belarusPartnerLogo from '@assets/appOfflineImages/belarus/partner/Минобр.png';
+export const blParterLogo=belarusPartnerLogo;
+import belarusSponserLogo from '@assets/appOfflineImages/belarus/sponser/Минздрав.png';
+export const blSponserLogo=belarusSponserLogo;
+import krygistanPartnerLogo from '@assets/appOfflineImages/krygystan/partner/LogoMoESKGWhiteSide.png';
```

Comment:
Let's rename the folder to kyrgyzstan
and krygistanPartnerLogo to kyrgyzstanPartnerLogo